modules/ROOT/nav.adoc

```diff
@@ -800,6 +800,7 @@ include::cli:partial$cbcli/nav.adoc[]
     *** xref:rest-api:rest-fts-advanced.adoc[Advanced]
     *** xref:rest-api:rest-fts-statistics.adoc[Search Statistics]
     *** xref:rest-api:rest-fts-query.adoc[Active Queries]
+    *** xref:rest-api:rest-fts-partition-file-transfer.adoc[Rebalance with File Transfer]
```

Comment:
Rebalance with File Transfer => doesn't sound the best wording in my limited knowledge. (not a native speaker)

File Transfer Based Rebalance  Or
Rebalance based on file transfer  Or

may sound better perhaps?

Reply:
Ok. I'll use Rebalance Based on File Transfer.